usage/rest-server/src/main/java/brooklyn/rest/security/PasswordHasher.java

```diff
@@ -24,8 +24,9 @@
 
 public class PasswordHasher {
     public static String sha256(String salt, String password) {
-        byte[] salted = Charsets.UTF_8.encode((salt == null ? "" : salt) + password).array();
-        HashCode hash = Hashing.sha256().hashBytes(salted);
+        if (salt == null) salt = "";
+        byte[] bytes = (salt + password).getBytes(Charsets.UTF_8);
```

Comment:
Appreciate this is existing behaviour but calling `sha256(null, null)` will mean we hash "null" (so always `74234e98afe7498fb5daf1f36ac2d78acc339464f950703b8c019892f982b90b`). I think this is a bit surprising and potentially revealing. Should we throw if `password` is null?